projects/plugins/jetpack/extensions/blocks/premium-content/subscriber-view/edit.js

```diff
@@ -48,6 +50,10 @@ export default compose( [
 			// @ts-ignore difficult to type with JSDoc
 			hasInnerBlocks: !! select( 'core/block-editor' ).getBlocksByClientId( props.clientId )[ 0 ]
 				.innerBlocks.length,
+			allowedInnerBlocks: select( 'core/blocks' )
+				.getBlockTypes()
+				.filter( blockType => blockType.name !== name )
+				.map( block => block.name ),
```

Comment:
This logic is pretty much duplicated in logged-out-view/edit.js, maybe we could generalize this into a separate helper/util/shared so that we could import it here and potentially in any other place without needing to copy pasta. 
WDYT?

Reply:
Sounds good, done! :)